pennylane_cirq/cirq_device.py

```diff
@@ -171,7 +152,16 @@ def __init__(self, wires, shots, qubits=None):
         "Toffoli": CirqOperation(lambda: cirq.TOFFOLI),
     }
 
-    _observable_map = {
+    _operation_map = {
+        **_base_operation_map,
+        **{
+            f"Adjoint({key})": CirqOperation(value.parametrization, adjoint=True)
+            for key, value in _base_operation_map.items()
+            if value and key[:8] != "Adjoint("
```

Comment:
What does `if value` do?

Reply:
Do you know if:
```python
CirqOperation(CirqOperation(lambda: cirq.ISWAP).parametrization, adjoint=True)
```
is the same as:
```python
CirqOperation(lambda: cirq.ISWAP_INV)
```
?

We then could remove all the `Adjoint` operations from `_base_operation_map` and remove the string check.